project/step07/lib/app.js

```diff
@@ -12,7 +14,9 @@ app.use(bodyParser.json());
 // TODO : call passeport serialize and deserializeUser
 module.exports = db => {
 
-  // TODO : use a passeport local strategy (use databse find method with selector)
+  // TODO : configure passeport here to use local strategy (this strategy use database find method with selector)
+  // Local strategy should be declared in user.js
```

Comment:
Il n'y a pas de **user.js** dans le step07, il faudrait supprimer ce commentaire.

Reply:
The file user.js should be created in this step